wqflask/tests/wqflask/show_trait/test_export_trait_data.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+from wqflask.show_trait.export_trait_data import dict_to_sorted_list,cmp_samples
```

Comment:
**issue:** Please try to stick to PEP 8 for this file. See: https://www.python.org/dev/peps/pep-0008/

**suggestion**: You can use a tool like autopep-8 and let it automatically fix (some) things for you.